### ISSUE TITLE
sort-package-json removes the last blank line when autofixing

### ISSUE THREAD

**Issue by EvgenyOrekhov** [kellyselden/eslint-plugin-json-files#39]
Before fixing:

```json
{
  "version": "1.0.0",
  "name": "foo"
}


```
^^^^ blank line at the end of the file

After fixing:

```json
{
  "name": "foo",
  "version": "1.0.0"
}
```

^^^^ no blank line at the end of the file

The last blank line should be preserved.

**Comment by canac** [comment]
@kellyselden Any chance of getting this fixed? I could put together a PR, but I'm not super familiar with ESLint. Adding an extra `\n` in the fixer seems like it should do the trick. Is there a better way to fix this?

**Comment by kellyselden** [comment]
If the file doesn't have a newline, we have to also not add one. Feel free to attempt a fix.

**Comment by kellyselden** [comment]
I started this in https://github.com/kellyselden/eslint-plugin-json-files/tree/final_newline, but didn't couldn't get both cases passing. If anyone wants to take this over.

**Comment by SimenB** [comment]
Fixed in #88 linked above, fwiw
